src/CatalystsList.ts

```diff
@@ -59,21 +59,21 @@ export = (
         }
     ],
     "goerli": [
+        {
```

Comment:
Not sure if this file should be commited as well.

Reply:
I think so, but I'm not 100% sure either.